src/ReportManager/EFBForm/index.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from 'react';
+
+import FormSchemaContextProvider from './FormSchemaContext';
+import Section from './fields/Section';
+
+const EFBForm = ({ schema, formData: initialFormData = {}, isDisabled, className, onChange, onError }) => {
+  const [formData, setFormData] = useState(initialFormData);
```

Comment:
What's the purpose of having the form data controlled here? WHy not to simply call `onChange` and let the parent keep the changes?

Reply:
I haven't get there yet, after addressing the feedback of this round I will return to take a look to the behavior of the form component 🫡